[BUG] `mvn test -Dbuildver=330` fails

**Describe the bug**
Running `mvn clean test -Dbuildver=330` fails with the error:

```
[INFO]      [exec] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.8:get (default-cli) on project rapids-4-spark_2.12: Couldn't download artifact: com.nvidia:rapids-4-spark-aggregator_2.12:jar:spark330:22.06.0-SNAPSHOT was not found in https://oss.sonatype.org/content/repositories/snapshots during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of snapshots-repo has elapsed or updates are forced
```

The workaround is to install first:

``` bash
mvn install -Dbuildver=330 -DskipTests
mvn test -Dbuildver=330
```

**Steps/Code to reproduce bug**
`mvn clean test -Dbuildver=330`

**Expected behavior**
Should work

**Environment details (please complete the following information)**
Not relevant. Reproduced on multiple environments.

**Additional context**
None

--- a/tests/README.md
+++ b/tests/README.md
@@ -16,8 +16,9 @@ and the code is in the `com.nvidia.spark.rapids.tests.mortgage` package.
 Unit tests exist in the [tests]() directory. This is unconventional and is done so we can run the 
 tests on the final shaded version of the plugin. It also helps with how we collect code coverage. 
 
-The `tests` module depends on the `aggregator` module which is shading the dependencies.
-Should first `mvn install` the aggregator jar to local Maven repository due to a limitation of the shading system.
+The `tests` module depends on the `aggregator` module which shades dependencies. When running the
+tests via `mvn test`, make sure to do an install via `mvn install` for the aggregator jar to the
+local maven repository.
 The steps to run the unit tests:
 ```bash
 cd <root-path-of-spark-rapids>

--- a/tests/README.md
+++ b/tests/README.md
@@ -16,7 +16,15 @@ and the code is in the `com.nvidia.spark.rapids.tests.mortgage` package.
 Unit tests exist in the [tests]() directory. This is unconventional and is done so we can run the 
 tests on the final shaded version of the plugin. It also helps with how we collect code coverage. 
 
-Use Maven to run the unit tests via `mvn test`.
+The `tests` module depends on the `aggregator` module which is shading the dependencies.
+Should first `mvn install` the aggregator jar to local Maven repository due to a limitation of the shading system.
+The steps to run the unit tests:
+```bash
+cd <root-path-of-spark-rapids>
+mvn clean install
+cd tests
+mvn test
+```
 
 To run targeted Scala tests append `-DwildcardSuites=<comma separated list of wildcard suite
  names to execute>` to the above command. 

